dhall/src/Dhall.hs

```diff
@@ -1324,7 +1324,7 @@ instance Inject a => Inject (Seq a) where
       where
         embedOut xs = ListLit (Just declaredIn) (fmap embedIn xs)
 
-        declaredOut = App List declaredIn
+        declaredOut = App List declaredIn -- TODO?
```

Comment:
I wasn't sure whether I'd need to change anything here.

Do we have tests for these instances BTW?

Reply:
There is a `Dhall.Test.Dhall` module in the test suite for testing this module although I don't remember if it covers this code or not

Either way, I don't believe you need to change anything here